### PR TITLE
Bug 1936443: Revert "baremetal: send full ignition to masters"

### PR DESCRIPTION
**Pull request by hardys**
This doesn't work for IPI baremetal deployments driven via hive,
because there are firewall rules that prevent access to the
bootstrap MCS from the pod running the installer.

This was implemented in:
https://github.com/openshift/installer/pull/4427

But we ran into problems making the same approach work for
worker machines ref:
https://github.com/openshift/installer/pull/4456

We're now looking at other approaches to resolve the
network-config requirements driving that work, so
switching back to the pointer config for masters seems
reasonable, particularly given this issue discovered for
hive deployments.

Conflicts:
  pkg/tfvars/baremetal/baremetal.go

This reverts commit 98dc38170a66d3ede7513fc01e3ccce4ff8dd93d.

### CODE REVIEW

**Comment by hardys** [comment]
WIP pending testing and more details via a bz (since we'll likely want to backport this to 4.7)

/cc @stbenjam @kirankt

**Comment by hardys** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1936443

**Comment by hardys** [comment]
/retitle Bug 1936443: Revert "baremetal: send full ignition to masters"

**Comment by stbenjam** [comment]
/approve
/assign @kirankt

@kirankt Could you please have a look and leave a lgtm if it's ok? Thanks!

**Review by kirankt — APPROVED** [review]
/lgtm

**Comment by blkingrh** [comment]
The ACM team would like to test with this when it is available. Is this expected in 4.7.3? It doesn't look like the PR merged correctly ^^^^. Is there a nightly build we can test with now, or is it still to come?

**Comment by stbenjam** [comment]
@blkingrh It's in 4.8 nightlies now, not sure which z-Stream it will make in 4.7.

@hardys @kirankt Would one of you be able to manually cherry-pick this to release-4.7?

**Comment by kirankt** [comment]
> @blkingrh It's in 4.8 nightlies now, not sure which z-Stream it will make in 4.7.
> 
> @hardys @kirankt Would one of you be able to manually cherry-pick this to release-4.7?

@stbenjam I'll do the manual cherry pick. Thanks.

**Comment by blkingrh** [comment]
Thank you for the updates, @kirankt @stbenjam .  Could you add a comment here when it is in a 4.7 nightly so we can know which one to pick up?

**Comment by blkingrh** [comment]
@kirankt @stbenjam Just checking back to see if this is in a 4.7 nightly or if there is a rough ETA on that? Thanks.

**Comment by kirankt** [comment]
@blkingrh Its not merged yet. I'm waiting on 4.8 revert to get verified by QA before the 4.7 backport can move along... I'll update you when it shows up in the nightlies... Thanks.

**Comment by blkingrh** [comment]
Just checking in to see if there is an ETA on when this will be available in 4.7? Thanks.

**Comment by anthonyfister** [comment]
Tagging so I can get updates as well. 
@kirankt There are several ACM zStreams that are dependent upon this fix getting back ported to 4.7.

**Comment by kirankt** [comment]
This 4.8 bug still need to be verified by QA before 4.7 gets any attention. Unfortunately this out of my control, but I'll make sure to bug the QA folks to look into the issue.

**Comment by blkingrh** [comment]
@kirankt any new news on the QA verification? Thanks.

**Comment by kirankt** [comment]
> @kirankt any new news on the QA verification? Thanks.

@blkingrh AFAIK, its still in the same state. I've reached out to QA to see when they can start verifying this bug, so that the backports can make progress... Thanks.

**Comment by blkingrh** [comment]
@kirankt We have a major customer who we believe is going to be looking for this support in the near future. Can you put us in touch with the QA lead so that we can understand whether or not this is in plan and what the timeline is?

**Comment by kirankt** [comment]
> @kirankt We have a major customer who we believe is going to be looking for this support in the near future. Can you put us in touch with the QA lead so that we can understand whether or not this is in plan and what the timeline is?

Hi @blkingrh . The QA contact is in the bugzilla link. Let me update it and make it Urgent, especially if there is a customer waiting for it...

**Comment by evelinec** [comment]
The BZ bug was verified by Alexander Chuzhoy:
I verified https://bugzilla.redhat.com/show_bug.cgi?id=1936443
:bugzilla: #1936443 [VERIFIED] Hive based OCP IPI baremetal installation fails to connect to API VIP port 22623 – high/:warning:urgent in Installer for 4.8/4.8.0

**Comment by kirankt** [comment]
The 4.7 bug is now in verified state and should move forward with the appropriate labels.

**Comment by blkingrh** [comment]
Thank you @kirankt . When do you anticipate that we can pick this up in a 4.7 build?

**Comment by evelinec** [comment]
Can you let us know if the 4.7 fix is in any build that we can verify?

**Comment by kirankt** [comment]
@evelinec , @blkingrh Please pose the questions regarding when the patch gets into a release in the appropriate bugzillas. The right people will see and respond to it. On github, as a developer, I can only work on the fix and see it off to completion. I honestly don't know the workings of the release process and github isn't the place to pose questions related to the release. Please comment/reply on the BZ.

**Comment by evelinec** [comment]
Thank you @kirankt. 

Just fyi for others, here's the 4.7 BZ, https://bugzilla.redhat.com/show_bug.cgi?id=1940275. I requested some info there.

**Pull request merged**
